src/dlgTriggerEditor.cpp

```diff
@@ -417,67 +354,6 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     showDebugAreaAction->setToolTip(tr("Activates Debug Messages -> system will be *MUCH* slower"));
     connect( showDebugAreaAction, SIGNAL(triggered()), this, SLOT( slot_debug_mode() ));
 
-    QAction * addTriggerMenuAction = new QAction( QIcon( QStringLiteral( ":/icons/tools-wizard.png" ) ), tr("Triggers"), this);
-    viewTriggerAction->setStatusTip(tr("Add Trigger"));
-    connect(addTriggerMenuAction, SIGNAL(triggered()), this, SLOT(slot_addTrigger()));
-
-    QAction * addAliasMenuAction = new QAction( QIcon( QStringLiteral( ":/icons/system-users.png" ) ), tr("Aliases"), this);
-    addAliasMenuAction->setStatusTip(tr("Add Alias"));
-    addAliasMenuAction->setEnabled( true );
-    connect( addAliasMenuAction, SIGNAL(triggered()), this, SLOT( slot_addAlias()));
-
-    QAction * addTimersMenuAction = new QAction( QIcon( QStringLiteral( ":/icons/chronometer.png" ) ), tr("Timers"), this);
-    addTimersMenuAction->setStatusTip(tr("Add Timer"));
-    addTimersMenuAction->setEnabled( true );
-    connect( addTimersMenuAction, SIGNAL(triggered()), this, SLOT( slot_addTimer()));
-
-    QAction * addVarsMenuAction = new QAction( QIcon( QStringLiteral( ":/icons/chronometer.png" ) ), tr("Variables"), this);
```

Comment:
Yeah, a side effect of using a QGridLayout and possibly cutting out separate nested layouts for the top and bottom halves of (the coloured bits of) vars_main_are.ui - I think the hidden-variable widgets are using 2 columns in the second (bottom) with the other two using one each, in the bottom row and the "Name:" label is using one on the top row with the QLineEdit display for the value of that taking up 3 columns...

I think I will have to resort (revert?) to two separate `QHorizontalLayouts`, one for each row, inside a `QVerticalLayout` for those rows.